frb_codegen/src/ir/field.rs

```diff
@@ -1,9 +1,46 @@
-use crate::ir::*;
+use crate::{ir::*, parser::DefaultValues};
 
 #[derive(Debug, Clone)]
 pub struct IrField {
     pub ty: IrType,
     pub name: IrIdent,
     pub is_final: bool,
     pub comments: Vec<IrComment>,
+    pub default: Option<DefaultValues>,
+}
+
+impl IrField {
+    #[inline]
+    pub fn required_modifier(&self) -> &str {
+        if self.default.is_some() {
+            ""
+        } else {
+            self.ty.dart_required_modifier()
+        }
+    }
+    pub fn field_default(&self, freezed: bool) -> String {
+        self.default
+            .as_ref()
+            .map(|default| {
+                let default = match default {
```

Comment:
Not sure whether `default` will be a keyword in the future? https://users.rust-lang.org/t/what-is-the-default-keyword-means-in-rust/57448

Reply:
IIRC it was meant to be used for specialization, as in `default fn`. I'm welcome to suggestions for replacements however.

Reply:
Then maybe we just keep it like this. If in the future it becomes a keyword, we can `r#` at that time easily.